conf/modules/tlsf.xml

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="tlsf">
+  <doc>
+    <description>
+      Dynamic memory allocator
+      Can be used as a replacement of malloc with a O(1) allocation time
```

Comment:
This is modified version, so I should have made a dedicated repository for this, and I though it was not much needed for only two files.